.github/workflows/ci.yaml

```diff
@@ -355,6 +355,15 @@ jobs:
           cd tests/integration/
           npx jest --ci --color read-write-hook
           helm -n integration-tests uninstall test-scan update-category update-severity
+      - name: "Hooks (Read-Only) Integration Tests"
+        run: |
+          helm -n integration-tests install test-scan ./scanner/test-scan/ --set="parserImage.tag=sha-$(git rev-parse --short HEAD)"
+          helm -n integration-tests install http-webhook ./demo-apps/http-webhook
+          WEB_HOOK_IP=$(kubectl get -n integration-tests service/http-webhook -o jsonpath='{.spec.clusterIP}')
+          helm -n integration-tests install ro-hook ./hooks/generic-webhook/ --set="webhookUrl=http://$WEB_HOOK_IP/hallo-welt"
```

Comment:
You should be able to use the dns inside the cluster to resolve the ip address instead of getting manually beforehand: https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/

As the domain name you should just be able to use the service name:
`--set="webhookUrl=http://http-webhook/hallo-welt"`